### ISSUE TITLE
Mono-Crag Instances

### ISSUE THREAD

**Issue by dorthrithil** [LocalCrag/LocalCragApp#97]
Some instances will have only a single crag. They should start in this crag and navigation etc must be made easier for this.

**Comment by dorthrithil** [comment]
The following scenarios should be possible:

Region -> Crag -> Sector -> Area/Block -> Line
Crag -> Sector -> Area/Block -> Line
Crag -> Sector/Area -> Line
Franchise -> Gym -> Area -> Line
Gym -> Area -> Line

Naming doesn't matter at first, this can be solved via translations. Ideally there is an option in transloco to overwrite just specific keys of a language. This way we don't run in a large number of languages if we actually want to support multiple languages (then we would need "german gym", "german outdoor", "german franchise", "english gym", ...)

To make our life easier, I would just create a dummy crag/sector for mono-crag / mono-sector instances. The urls wouldn't be perfect then, but this doesnt really matter.

**Comment by dorthrithil** [comment]
Could maybe be done with a fallback language. So the german gym language would just have german as fallback and then only keys that should overwrite standard translations.

**Comment by dorthrithil** [comment]
> Could maybe be done with a fallback language. So the german gym language would just have german as fallback and then only keys that should overwrite standard translations.

@BlobbyBob When you start the gym language, maybe this could be an approach

**Comment by BlobbyBob** [comment]
Yeah, thanks for the ideas and the write-up. I will tackle separate colors for each line first, and then this issue will be tackled.
